### PR TITLE
Update to RestEasy 4.5.9.Final

### PR DESCRIPTION
**Pull request by sberyozkin**
CC @ia3andy @FroMage

### CODE REVIEW

File: integration-tests/pom.xml [hunk 1]
```diff
@@ -141,6 +141,8 @@
         <module>reactive-messaging-kafka</module>
         <module>reactive-messaging-http</module>
         <module>rest-client</module>
+        <module>rest-client-exception</module>
+        <module>rest-client-exception-original-behavior</module>
```
Comment (gsmet) [inline integration-tests/pom.xml:145]: Does it really make sense to add two new integration tests modules for that? Can't we test that in tests in the `deployment` module?

**Review by gsmet — CHANGES_REQUESTED** [review]

File: integration-tests/pom.xml (hunk 1 above)
Comment (sberyozkin) [reply integration-tests/pom.xml:145]: @gsmet Sure, I can do it

**Review by ia3andy — APPROVED** [review]
Awesome!

**Comment by gsmet** [comment]
@sberyozkin so I'm not sure about this one? Should it be backported? I have no idea what it is supposed to fix and how risky it is to upgrade?

**Comment by sberyozkin** [comment]
Hi @gsmet I apologize for the confusion, I meant to set the milestone to `1.12` but chose a wrong one, all fixed now, thanks

File: integration-tests/pom.xml (hunk 1 above)
Comment (sberyozkin) [reply integration-tests/pom.xml]: @gsmet done :-), let me know please if you'd like some more tweaks there

**Review by gsmet — APPROVED** [review]

**Comment by sberyozkin** [comment]
@ia3andy @gsmet thanks

**Pull request merged**